src/components/v5/common/ActionSidebar/partials/forms/EditTeamForm/utils.tsx

```diff
@@ -33,17 +37,34 @@ export const editTeamDescriptionMetadataGetter: DescriptionMetadataGetter<
 
 export const getEditDomainPayload = (
   colony: Colony,
-  values: EditTeamFormValues,
-  selectedDomain: Domain,
-) => ({
-  colonyAddress: colony.colonyAddress,
-  colonyName: colony.name,
-  domainName: values.teamName,
-  domainColor: values.domainColor,
-  domainPurpose: values.domainPurpose,
-  domain: selectedDomain,
-  annotationMessage: values.description,
-  customActionTitle: values.title,
-  motionDomainId: values.createdIn,
-  isCreateDomain: false,
-});
+  {
+    description,
+    domainColor,
+    domainPurpose,
+    teamName,
+    decisionMethod: decisionMethodValue,
+  }: EditTeamFormValues,
+  domain: Domain,
+) => {
+  const payload: ActionPayloadWithoutCustomTitle<ActionTypes.ACTION_DOMAIN_EDIT> =
```

Comment:
This shouldn't omit custom title

Reply:
The title is added by parent transform here https://github.com/JoinColony/colonyCDapp/blob/f643066e6b3d1ff80d3b29804a4e06685ebef891/src/components/v5/common/ActionSidebar/hooks/useActionFormProps.ts#L40

I have omitted it as it's required in sagas but not mandatory in transforms for each of the forms (in fact it would be overwritten by parent transform). I can make it optional but I think it's better to have it omitted here to avoid confusion. You can still get the original payload type by using `ActionPayload` type.